packages/cli/src/environments.ee/source-control/source-control-export.service.ee.ts

```diff
@@ -305,14 +305,25 @@ export class SourceControlExportService {
 
 	async exportTagsToWorkFolder(context: SourceControlContext): Promise<ExportResult> {
 		try {
+			const fileName = path.join(this.gitFolder, SOURCE_CONTROL_TAGS_EXPORT_FILE);
 			sourceControlFoldersExistCheck([this.gitFolder]);
 			const tags = await this.tagRepository.find();
-			// do not export empty tags
+
 			if (tags.length === 0) {
+				await fsWriteFile(
+					path.join(this.gitFolder, SOURCE_CONTROL_TAGS_EXPORT_FILE),
+					JSON.stringify({ tags: [], mappings: [] }, null, 2),
+				);
```

Comment:
Avoids doing the `path.join` again:

```suggestion
				await fsWriteFile(
					fileName,
					JSON.stringify({ tags: [], mappings: [] }, null, 2),
				);
```